logistic_regression.py

```diff
@@ -0,0 +1,45 @@
+import math
+import pandas as pd
+import statsmodels.api as sm
+import matplotlib.pyplot as plt
+
+# Load data on loans
+loans = pd.read_csv("loans_clean.csv")
+
+# Create boolean column for interest rate < 12%
+# I'm used to defining functions out like this, but there's probably something quicker for pandas
+def less_than_12(rate):
+	if rate < 0.12:
+		return 1
+	else:
+		return 0
+
+loans['IR_TF'] = loans['Interest.Rate'].map(less_than_12)
```

Comment:
we can do this MUCH more easily.

`loans['IR_TF'] = loans['Interest.Rate'] < .12`